utils/axiosSetp.mjs

```diff
@@ -0,0 +1,9 @@
+import { create } from 'middleware-axios';
+
+// create wrapped instance of axios to use normal axios instance
+const axios_api = create({
```

Comment:
`import { create } from 'middleware-axios';

/**
 * Axios middleware to attach Axios instance to request object.
 *
 * @param {object} req - The Express request object.
 * @param {object} res - The Express response object.
 * @param {function} next - The next middleware function in the stack.
 */
const axiosMiddleware = (req, res, next) => {
    const protocol = req.protocol;
    const host = req.get('host');
    const baseURL = `${protocol}://${host}`;
    
    // create wrapped instance of axios to use normal axios instance
    req.axios = create({
        baseURL: baseURL,
        timeout: 5000, // Set a timeout value if needed
        headers: {
            'Content-Type': 'application/json',
            // You can add other default headers here if needed
        },
    });
    next();
};

export default axiosMiddleware;`

Reply:
@psweeting1 - Not sure what you want me to do with this comment. Copying and pasting this in, throws a bunch of errors. I'm using the [middelware_axios ](https://www.npmjs.com/package/middleware-axios), which I think does the above already hence implementing that way, .

Reply:
Sorry I'll need the errors in order to help you.